sdk/core/azure-core/inc/nullable.hpp

```diff
@@ -103,6 +103,16 @@ namespace Azure { namespace Core {
       return *this;
     }
 
+    template <
+        class U = T,
+        typename std::enable_if<
```

Comment:
This constraint doesn't make sense because U isn't in the interface of operator= here.